content/docs/06-integrations/00-ubuntu.md

```diff
@@ -166,21 +168,29 @@ For more information see the [Ubuntu Advantage for Infrastructure](https://ubunt
+
+| **Service**  | **Options**    |  **Description**                                                                                                                                                                                                                                                                                                                                                                                |
+| --------------- | -------------- | ------------------------------------------------------------------ | ------------------------------ |
+|  **Token**      |                                    | Enter the token key in the text box. <br /> e.g.: C13RaHQDqgvvG3Ys|
+| **ESM-infra** | enable <br /> <br /> <br /> <br /> <br /> disable | Continue to receive security updates for the Ubuntu base OS, critical software <br /> packages and infrastructure components with Extended Security Maintenance (ESM). <br />  ESM provides five additional years of security maintenance, enabling an organization's <br />continuous vulnerability management. <br /> <br />  Disable the ESM repository. Do not receive security updates for Ubuntu Base OS etc. |
+|||||
+|**Livepatch**| enable <br /> <br /> <br /> <br />  disable| Livepatch eliminates the need for unplanned maintenance windows <br /> for high and critical severity kernel vulnerabilities by patching the Linux <br /> kernel while the system runs. <br /> <br /> Do not enable/manage live kernel patching. The Livepatch service will be disabled. |
+|||||
+| **FIPS** | enable <br /> <br /> <br /> <br /> disable | Federal Information Processing Standards (FIPS) 140 validated cryptography for  <br /> Linux workloads on Ubuntu. Install, configure, and enable <br /> FIPS 140 certified modules.  <br /><br /> Do not have FIPS 140 validated cryptography for Linux workloads on Ubuntu.|
+|||||
+| **FIPS-updates** | enable <br /> <br /> <br /> disable | The option enables the FIPS-UPDATES. It installs the updated FIPS modules sets <br />it in FIPS mode. <br /><br /> Do not have FIPS 140 validated cryptography for Linux workloads on Ubuntu.|
+|||||
+| **CC-EAL** | enable <br /> <br /> <br /> <br /> <br />  disable | After the completion of a Common Criteria (CC) security evaluation, a grade is given <br /> indicating the level the system was tested. Common criteria evaluated configuration is <br /> currently available for Ubuntu 16.04.4 LTS (Server) and Ubuntu 18.04.4 LTS (Server). <br /> The option shows as disabled for Ubuntu 20 & 20+ as per this documentation. <br /> <br /> Do not make enable and install the CC artifacts. |

```

Comment:
"as per this documentation" - is there a link on Canonical website to reference?
"Do not make enable and install the CC artifacts." - slight word change of "make enable"?